src/pages/HomePage.tsx

```diff
@@ -30,10 +30,10 @@ const useStyles = makeStyles({
 
 const HomePage: FC<IProps & RouteComponentProps<IRouteProps>> = ({ match }) => {
   const classes = useStyles();
-  const id = match.params.id ? parseInt(match.params.id) : null;
   const store: RootStore = useStore();
+  const initialId = store?.newsFlashCollection[0]?.id;
```

Comment:
`newsFlashCollection` can be an empty array

Reply:
ok
is it a statement or a comment that need to be resolved ?

Reply:
Needs to be resolved, e.i. Make the code safe for all cases